INSTALL.sh

```diff
@@ -1,19 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Let's move all of the things in the `INSTALL.sh` script to the invoke script as well.